src/main/java/chronologer/command/Command.java

```diff
@@ -28,7 +28,7 @@
      */
     public boolean isIndexValid(Integer indexOfTask, Integer currentSizeOfTasks) throws ChronologerException {
         if (indexOfTask < 0 || indexOfTask > (currentSizeOfTasks - 1)) {
-            UiTemporary.printOutput(ChronologerException.invalidIndex());
+            UiMessageHandler.outputMessage(ChronologerException.invalidIndex());
             throw new ChronologerException(ChronologerException.invalidIndex());
```

Comment:
got some mistakes like this around, ChronologerException already calls UiMessageHandler.outputMessage() now so this could be redundant

Reply:
Cleared with new pull request!